backend/service/k8s/deployment.go

```diff
@@ -63,9 +63,9 @@ func (s *svc) UpdateDeployment(ctx context.Context, clientset, cluster, namespac
 	}
 
 	newDeployment := oldDeployment.DeepCopy()
-	mergeLabelsAndAnnotations(newDeployment, fields)
+	mergeDeploymentLabelsAndAnnotations(newDeployment, fields)
 
-	patchBytes, err := generateDeploymentStrategicPatch(oldDeployment, newDeployment)
```

Comment:
can you delete the `generateDeploymentStrategicPatch` function as well?

Reply:
removed it and its tests